tests/cli/test_push.py

```diff
@@ -1151,53 +1150,6 @@ def test_push_sync_keymap(
     assert listdata(resp_city, '_id', 'id', 'name', 'country')[0] == (city_id, 1, 'Vilnius', {'_id': country_id})
 
 
-@pytest.mark.skip("Private now sends warning that model has been skipped syncing rather throwing exception")
```

Comment:
Why was this test removed? It does not seem to be duplicated.

Reply:
Removed because it was testing `UnauthorizedKeymapSync`, which is not used anywhere. It's dead and skipped test.